src/models/userDao.js

```diff
@@ -0,0 +1,46 @@
+const database = require("../utils/database")
+
+
+  // 요청 이메일 존재 여부 확인
+const select = async(email) => {
+  try{
+    const result = await database.appDataSource.query(
+      `
+      select * from users
+      where email = ?
+      `, [email]
+    )
+      return result;
+  }catch(err){
+    console.log(err);
+    const error = new Error();
+    error.statusCode = 500;
+    throw error;
+  }
+}
+
```

Comment:
```suggestion
  // 요청 이메일 존재 여부 확인
const selectUserDataByEmail = async(email) => {
  try{
    const result = await database.appDataSource.query(
      `
      select id from users
      where email = ?
      `, [email]
    )
      return result;
  } catch(err) {
    console.log(err);
    const error = new Error();
    error.statusCode = 500;
    throw error;
  }
}

```

1. 함수의 이름은 어떤 동작을 정확히 수행하는지 구체적으로 명시되어 있어야 합니다. select는 어떤 과정이 진행되는지 알기 어려운 함수명이으로 보다 구체적이고 명확한 함수명으로 네이밍 해주세요.
2. `*`를 활용해 모든 데이터를 불러오는 것은 좋은 방법이 아닙니다. 필요한 컬럼만 선정해서 SELECT 해주세요.
3. 에러를 catch해서 500을 던지는 핸들링은, 에러핸들링이 안되는 것과 동일합니다. 게다가 메세지도 new Error()로 빈 메세지를 보내면 아무런 디버깅이 안됩니다. 적절한 메세지를 작성하시거나 아예 try-catch문을 제거해 주세요